build/js/live-editor.output_pjs.js

```diff
@@ -1602,10 +1614,38 @@ window.PJSOutput = Backbone.View.extend({
             // but PImage may be mutable, so that might not work.
             this.imageCache[file] = img;
         }.bind(this));
-    },
+        
+        _.each(sounds, function(file) {
```

Comment:
We may have one other visual type of loader, the leaf, but the throbber is the standard visual. I think we could just put text underneath the throbber it to say what state its on ("Loading code, checking code, loading 3 images, loading 4 sounds, etc"). But that doesn't need to be done here.